CHANGELOG.md

```diff
@@ -1,43 +1,94 @@
+- Added <kbd>t</kbd> to toggle between streams & list of all topics in selected stream
+- Added <kbd>m</kbd> to mute/unmute selected stream, with confirmation popup
+- Added <kbd>i</kbd> to show further information of selected message (including reactions)
 - Use <kbd>?</kbd> to show *and* hide help menu (<kbd>esc</kbd> continues to exit)
-- Add additional shortcut key for sending messages (<kbd>ctrl</kbd>+<kbd>d</kbd>)
+- Add additional shortcut key for sending messages (<kbd>ctrl</kbd>+<kbd>d</kbd>) (#176)
+- Add hint in footer of how to copy text, if attempt to select text (with mouse) in terminal
+- Stream search is now case-insensitive
+- Add command-line option to check version (#280)
 - Allow <kbd>f</kbd> and <kbd>P</kbd> shortcut keys to work from side panels (narrow starred & private messages)
 - Warn on startup, if specified theme is lacking current required styles
 - Upon unexpected crash, exit cleanly and log traceback to `zulip-terminal-tracebacks.log`
-- Add <kbd>z</kbd> hot/shortcut key to toggle between stream/topic or all-private/specific-person narrows
 
 ### Visual improvements
 - Right-align unread-counts in left & right panels (as in webapp)
 - Remove intrusive flashing cursor in side panels
-- Truncate text in left & right panels cleanly with '..', avoiding text overflow
-- Show user status as `active` (Green), `idle` (Yellow) or `offline` (White) using different colors.
+- Truncate text in left & right panels cleanly with '…', avoiding text overflow (#237)
+- Show users' status as `active` (Green), `idle` (Yellow) or `offline` (White) using different colors.
+- Update user list every minute, not just at startup (#330)
+- Sort user list alphabetically, case-independently (#264)
 - Further improvement/reordering of shortcut keys in README & help menu (<kbd>?</kbd>)
-- Improve styling of help menu, and how the menu scales with application width & height
+- Improve styling of help menu, and how it menu scales with application width & height (#297, #307)
 - Make stream icons bold and correct background color
+- Themes improved (#254)
+- Support terminal transparency in default theme (#377)
+- Clarify that <kbd>ctrl</kbd>+<kbd>c</kbd> is the intended way to exit/quit (#208)
+- Mark edited messages with 'EDITED' at their top-left
+- Show current narrow with search box, above current-recipient indicator for message (#201, #340)
+- Hide current-recipient indicator when fully narrowed (in topic or conversation)
+- Improve rendering of quoted text with indents
+- Display custom emoji text-equivalents in messages and reactions (#238)
+- Readline keys are now listed in the online help
+- Change main widget styles, to clearly separate three columns, with bold top-lines
+- Minimize space taken up by search boxes & display shortcut keys for each
+- Remove bullets & display shortcut keys for for all messages, private messages & starred messages
+- Style private message header like webapp (You and ...)
 
```

Comment:
Would we want to add , `Extended Left column 2 spaces more to the right while shrinking User Column to compensate`?

Reply:
This didn't feel like too much of an important change to note; it could affect the UI (spacing for names etc), but the ellipsis change gives more space, and previously the text just over-ran in 0.3.1, I think.

Reply:
Sure. I was not aware of the older behaviour in 0.3.1. Agreed that this seems to be the more highlighted change to be mentioned here.